angular/src/components/settings/vault-timeout-input.component.ts

```diff
@@ -132,10 +136,18 @@ export class VaultTimeoutInputComponent implements ControlValueAccessor, Validat
       return { policyError: true };
     }
 
+    if (!this.getValidMinutesInput()) {
+      return { minTimeoutError: true };
+    }
+
     return null;
   }
 
   registerOnValidatorChange(fn: () => void): void {
     this.validatorChange = fn;
   }
+
+  getValidMinutesInput(): boolean {
```

Comment:
I don't think this name is very clear, maybe use a proper getter and then call it something like:

```suggestion
  get exceedsMinimumTimeout(): boolean {
```